.github/workflows/battlecode.yml

```diff
@@ -0,0 +1,37 @@
+name: Java CI
```

Comment:
This is a github action workflow file. its kind of like a hybrid between a configuration file and a script that tells on what event the action should be executed on. Then it runs the rest of the script on some github host computer. Every step is self docuemented with its name. Each run is just a shell command